docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Work with workbooks using the Excel JavaScript API
+description: ''
+ms.date: 11/26/2018
+---
+
+
+# Work with Workbooks using the Excel JavaScript API
+
+This article provides code samples that show how to perform common tasks with workbooks using the Excel JavaScript API. For the complete list of properties and methods that the **Workbook** object supports, see [Workbook Object (JavaScript API for Excel)](/javascript/api/excel/excel.workbook). This article also covers workbook-level actions performed through the [Application](/javascript/api/excel/excel.application) object.
+
+The workbook object is the entry point for your add-in to interact with Excel. It contains a [WorksheetCollection](/javascript/api/excel/excel.worksheetcollection) to add worksheets, navigate among them, and assign handlers to worksheet events. The article [Work with Worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md) describes how to access and edit worksheets.
+
+## Get active cell and selected range
+
+Workbooks maintain collections of worksheets, tables, PivotTables, and other data types through which Excel data is accessed and changed. They also have application-level information (more on that later). There are a pair of methods to directly get ranges the Excel user has just activated. The first is `getActiveCell()`. This method gets the active cell from the workbook as a [Range object](/javascript/api/excel/excel.range). The following example shows a call to `getActiveCell()`, followed by the cell's address being printed to the console.
+
+
+```js
+Excel.run(function (context) {
+    var activeCell = context.workbook.getActiveCell();
+	activeCell.load("address");
+
+    return context.sync().then(function () {
+		console.log("The active cell is " + activeCell.address);
+	});
+}).catch(errorHandlerFunction);
+```
+
+The second range access method is `getSelectedRange()`. This method returns the range currently selected by either a user or an add-in. The following example shows a call to `getSelectedRange()` that then sets the range's fill color to yellow.
+
+```js
+Excel.run(function(context) {
+	var range = context.workbook.getSelectedRange();
+	range.format.fill.color = "yellow";
+	return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+## Create workbook
+
+Your add-in can create a new workbook, separate from the Excel instance in which the add-in is currently running. The Excel object has the `createWorkbook` method for this purpose. When this method is called, the new workbook is immediately opened and displayed. This happens in a new Excel instance. Your add-in remains open and running with the previous workbook.
```

Comment:
Consider combining sentences #3 and 4 -> When this method is called, the new workbook is immediately opened and displayed in a new instance of Excel.